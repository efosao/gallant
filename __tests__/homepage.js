// Link.react.test.js
import React from 'react'
import Homepage from '../pages/index'
import renderer from 'react-test-renderer'

test('Test homepage', () => {
  const component = renderer.create(
    <Homepage />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()

  // manually trigger the callback
  // tree.props.onMouseEnter();
  // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();

  // manually trigger the callback
  // tree.props.onMouseLeave();
  // re-rendering
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
})
