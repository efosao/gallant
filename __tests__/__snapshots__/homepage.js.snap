// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test homepage 1`] = `
<div>
  Welcome to Next.js!
</div>
`;
